package.json

```
@@ -16,7 +16,7 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-countdown-circle-timer": "^2.5.4",
-    "react-datepicker": "^4.5.0",
+    "react-datepicker": "^4.6.0",
```

```suggestion
    "react-datepicker": "^4.5.0",
```
Please revert back to older version, unless its necessary.